### DISCUSSION TITLE
[Item] 以文章标题为笔记标题的笔记 Note outline with top item title

### DISCUSSION THREAD

**Discussion by LynnXie00** [windingwind/zotero-better-notes#404] [Note Templates]
Type: Item
Description: This template is to quickly generate an outline consisting of general headings like "purpose", "research questions" using article title as the note title. 
Screenshots: 
How to use
![Snipaste_2023-08-01_08-21-02](https://github.com/windingwind/zotero-better-notes/assets/30601091/776c49f9-000b-4cc0-868c-ed7dd7d60aa9)
![Snipaste_2023-08-01_08-21-16](https://github.com/windingwind/zotero-better-notes/assets/30601091/fde2967e-e62b-44ca-8427-40645020ebbf)
![Snipaste_2023-08-01_08-21-27](https://github.com/windingwind/zotero-better-notes/assets/30601091/f5a819f9-88b2-462b-b70a-57b4c13e64d2)

Template Contents:  

```
${await new Promise(async (r) => {
    let res = `<h1>${topItem.getField("title")}</h1>
<br>
<h2>💡Purpose</h2>
<br>
<h2>❓Research questions </h2>
<br>
<h2>📚Theory </h2>
<br>
<h2>❗Hypotheses</h2>
<br>
<h2>⛏️Method</h2>
<br>
<h3>Measures</h3>
<br>
<h3>Manipulate</h3>
<br>
<h3>Analysis technique</h3>
<br>
<h2>📉Model</h2>
<br>
<h2>📝Results</h2>
<br>
<h2>👍Contributions</h2>
<h3>Theoretical Contributions</h3>
<br>
<h3>Practical Contributions</h3>
<br>
<h2>❌Limitation</h2>
<br>
<h2>↪️Future Direction</h2>  `;
    r(res);
  })}





```

**Comment by rosie-law** [comment]
你好你好，我是个小白，我按照你的方法成功添加模版之后，无法插入使用，他会显示reference error，topitem is not defined，请问这时候应该怎么办呀，感谢！

**Reply by EscapeQuickly** [reply]
[Item] 以文章标题为笔记标题的笔记 Note outline with top item title   把模版名称改成这个就可以了
